server/src/test/java/org/elasticsearch/search/aggregations/bucket/histogram/NumericHistogramAggregatorTests.java

```diff
@@ -428,6 +428,53 @@ public void testExtendedBounds() throws Exception {
         }
     }
 
+    public void testHardBoundsOnDouble() throws Exception {
+        try (Directory dir = newDirectory(); RandomIndexWriter w = new RandomIndexWriter(random(), dir)) {
+            for (double value : new double[] { 3.2, -5, -4.5, 4.3 }) {
+                Document doc = new Document();
+                doc.add(new SortedNumericDocValuesField("field", NumericUtils.doubleToSortableLong(value)));
+                w.addDocument(doc);
+            }
+
+            HistogramAggregationBuilder aggBuilder = new HistogramAggregationBuilder("my_agg").field("field")
+                .interval(5)
+                .hardBounds(new DoubleBounds(0.0, 10.0));
+            MappedFieldType fieldType = new NumberFieldMapper.NumberFieldType("field", NumberFieldMapper.NumberType.DOUBLE);
+            try (IndexReader reader = w.getReader()) {
+                IndexSearcher searcher = new IndexSearcher(reader);
+                InternalHistogram histogram = searchAndReduce(searcher, new MatchAllDocsQuery(), aggBuilder, fieldType);
+                assertEquals(1, histogram.getBuckets().size());
+                assertEquals(0d, histogram.getBuckets().get(0).getKey());
+                assertEquals(2, histogram.getBuckets().get(0).getDocCount());
+                assertTrue(AggregationInspectionHelper.hasValue(histogram));
+            }
+        }
+    }
+
+    public void testHardBoundsOnHistogram() throws Exception {
```

Comment:
I'm confused, is this a different test than the one above it?  If so, I'm missing what's different.

Reply:
No, it's the same, I will remove it. Thanks for catching it! I was going to turn it this into a histogram field test before I realized that I cannot put it here and it needs to go into x-pack because of dependencies.